rule/struct_tag.go

```diff
@@ -134,22 +210,23 @@ func (w lintStructTagRule) checkTagNameIfNeed(tag *structtag.Tag) (string, bool)
 
 	// We concat the key and name as the mapping key here
 	// to allow the same tag name in different tag type.
-	key := tag.Key + ":" + tagName
-	if _, ok := w.usedTagName[key]; ok {
-		return fmt.Sprintf("duplicate tag name: '%s'", tagName), false
+	mapKey := tag.Key + ":" + tagName
+	if _, ok := ctx.usedTagName[mapKey]; ok {
+		return fmt.Sprintf(msgDuplicatedTagName, tagName), false
```

Comment:
The constant `msgDuplicatedTagName` is used only once and we can remove it. Perhaps the following is more readable:
```suggestion
		return fmt.Sprintf("duplicated tag name %q", tagName), false
```